### PR TITLE
 [xorstr] Update to 2020-11-29

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by JonLiu1993** [comment]
@tiqac ,Can you take a look at this error log：
````
curl failed to execute with exit code: 14336
Convert-Path: /Users/vagrant/Data/work/1/s/scripts/azure-pipelines/analyze-test-results.ps1:424
Line |
 424 |  … t_hash = build_test_results( Convert-Path "$logDir\$($triplet).xml" )
     |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path
     | '/Users/vagrant/Data/work/1/a/xml-results/x64-osx.xml' because
     | it does not exist.


##[error]PowerShell exited with code '1'.
##[error]PowerShell wrote one or more lines to the standard error stream.
##[error]Convert-Path: /Users/vagrant/Data/work/1/s/scripts/azure-pipelines/analyze-test-results.ps1:424
Line |
 424 |  … t_hash = build_test_results( Convert-Path "$logDir\$($triplet).xml" )
     |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path
     | '/Users/vagrant/Data/work/1/a/xml-results/x64-osx.xml' because
     | it does not exist.
````

**Comment by tiqac** [comment]
I don't think this error is related to changes in this PR

File: ports/xorstr/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "xorstr",
+  "version-string": "2020-11-29",
```
Comment (vicroms) [inline ports/xorstr/vcpkg.json:3]:
```suggestion
  "version-date": "2020-11-29",
```

**Review by vicroms — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Review by vicroms — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Thanks for the PR!

**Pull request merged**